Gini-iOS-SDK/GINIURLSessionDelegate.m

```diff
@@ -9,42 +9,55 @@
 #import "GINIURLSessionDelegate.h"
 
 @implementation GINIURLSessionDelegate {
-    NSString *_nsCertificatePath;
+    NSArray<NSString *> *_nsCertificatePaths;
```

Comment:
Not sure i agree with the `ns` prefix. And I don't think we usually put an underscore on private variables. But it also makes sense since it's not in the class definition. Just a comment...

Reply:
I just wanted to be consistant with the rest of the classes, I don't like neither but...